include/z64save.h

```diff
@@ -143,11 +169,11 @@ typedef struct {
     /* 0x13EA */ u16 unk_13EA; // also alpha type?
     /* 0x13EC */ u16 unk_13EC; // alpha type counter?
     /* 0x13EE */ u16 unk_13EE; // previous alpha type?
-    /* 0x13F0 */ s16 unk_13F0; // magic related
-    /* 0x13F2 */ s16 unk_13F2; // magic related
-    /* 0x13F4 */ s16 unk_13F4; // magic related
-    /* 0x13F6 */ s16 unk_13F6; // magic related
-    /* 0x13F8 */ s16 unk_13F8; // magic related
+    /* 0x13F0 */ s16 magicState; // determines magic meter behavior on each frame
+    /* 0x13F2 */ s16 prevMagicState; // used to resume the previous state after adding or filling magic
+    /* 0x13F4 */ s16 magicCapacityDrawn; // only differs from magicCapacity in a new save load, where magicCapacityDrawn is slowly stepped from 0 to magicCapacity
```

Comment:
I think you could just say "increased" instead of "stepped" here: it's a discrete variable, yes, but it looks continuous enough in-game, it's not like you see it increase 8 pixels, stop, then do it again, or whatever.

Reply:
What about "increased every frame"? I used step because it's increased every update function over time, so I wanted to capture that in a way

Reply:
Maybe describe the visuals a bit more? It's not clear just from here that it's explicitly to show the bar itself increasing in size.
```suggestion
    /* 0x13F4 */ s16 magicCapacityDrawn; // only differs from magicCapacity in a new save load, where magicCapacityDrawn is increased from 0 to magicCapacity to show the meter size growing to its actual maximum value.
```
?

Reply:
"gradually increased" (a.k.a. "stepped" I guess) makes it more clear to me there's an animation going on